website/content/release-notes/_index.md

```diff
@@ -9,8 +9,25 @@ weight: 7
 
 Action required if updating from 0.6.x:
 
+- MetalLB no longer does leader election, so you can delete a number
+  of k8s resources associated with that. This is just a cleanup,
+  nothing bad happens if you leave the resources orphaned in your
+  cluster. Depending on your installation method, some of these may
+  have already been cleaned up for you.
+  - `kubectl delete -nmetallb-system endpoints metallb-speaker`
```

Comment:
Nit: does kubectl allow flags with no spaces before the argument?

Reply:
Yup, it does! :)